radicle-cli/src/terminal/format.rs

```diff
@@ -17,15 +18,24 @@ pub fn node(node: &NodeId) -> String {
 }
 
 /// Format a git Oid.
-pub fn oid(oid: &radicle::git::Oid) -> String {
-    format!("{:.7}", oid)
+pub fn oid(oid: impl Into<radicle::git::Oid>) -> String {
+    format!("{:.7}", oid.into())
 }
 
 /// Format a COB id.
 pub fn cob(id: &ObjectId) -> String {
     format!("{:.11}", id.to_string())
 }
 
+/// Format a timestamp.
+pub fn timestamp(time: &Timestamp) -> String {
+    let fmt = timeago::Formatter::new();
+    let now = Timestamp::now();
+    let duration = time::Duration::from_secs(now.as_secs() - time.as_secs());
+
+    fmt.convert(duration)
+}
```

Comment:
What about a different method name like time_since(..) or similar?  The name and description appear a bit off from whats happening here.

Reply:
Nevermind, I see the greater context.